src/main/java/main/Employee.java

```diff
@@ -68,7 +68,7 @@ public String getFirstName() {
         return firstName;
     }
 
-        public String getLastName() {
+    public String getLastName() {
         return lastName;
     }
```

Comment:
Add to method equals verification on null;
Please read article https://javarush.ru/groups/posts/1340-peregruzka-metodov-equals-i-hashcode-v-java

Reply:
Constructor per convention should follow after fields

Reply:
remove unused comments